astroid/const.py

```diff
@@ -13,6 +13,7 @@ class Context(enum.Enum):
     Del = 3
 
 
-Load = Context.Load
-Store = Context.Store
-Del = Context.Del
+# TODO Remove in 3.0 in favor of Context
```

Comment:
Maybe we should add a Changelog entry for it?

Reply:
I think we can add it in the breaking changes for 3.0 directly. Except if you know how to raise a deprecation warning if a constant is imported ? I don't think it's possible